spec/ics-2-consensus-requirements/README.md

```diff
@@ -0,0 +1,124 @@
+---
+ics: 2
+title: Consensus Verification
+stage: proposal
+category: ibc-core
+requires: 23
+required-by: 3
+author: Juwoon Yun <joon@tendermint.com>, Christopher Goes <cwgoes@tendermint.com>
+created: 2019-02-25
+modified: 2019-03-05
+---
+
+## Synopsis
+
+Consensus verification requires the properties that the chains on the network are 
+expected to satisfy. The properties are needed for efficient and safe verification on the 
+higher level mechanics, such as connection and channel semantics. The algorithm who uses these 
+properties to verify another chain is referred as "lightclient", which is embedded on the chains.
+
+## Specification
+
+### Motivation
+
```

Comment:
I think we should talk about the full node - light client correspondence here - what we ultimately want is a cheap way for a blockchain to verify the results of the consensus process of another blockchain.